chore: change logs for e4.4.22 (#2200)
emqtt/emq-docs-cn: en_US/changes/changes-ee-4.4.md

# Releases

## e4.4.22

*Release Date: 2023-11-01*

### Enhancements

> - Enhanced logging capabilities through enabling the audit log module within the Dashboard.

Reviewer: ```suggestion
- Added the Audit Log feature to track important operation changes.
```

>   - To enable the audit log feature, please click on "General" -> "Audit Log" -> "Enable", modify the parameters and then click "Add".

Reviewer: ```suggestion
  - To enable the Audit Log, click **General** -> **Audit Log** -> **Enable** on Dashboard, modify the parameters, and click **Add**.
```

>   - Once enabled, all HTTP requests except `GET` requests, as well as the CLIs, will be recorded.

Reviewer: ```suggestion
  - Once enabled, all HTTP requests except `GET` and CLI executions will be recorded.
```

>   - By default, the most recent 5,000 audit log entries can be accessed through the dashboard, and the complete log file is stored in `data/audit` directory.

Reviewer: ```suggestion
  - The most recent 5,000 audit log entries are visible by default in the Dashboard, while the complete log file is stored in the `data/audit` directory.
```
